### ISSUE TITLE
Token list reached 20MB limit by JSDelivr

### ISSUE THREAD

**Issue by refi93** [solflare-wallet/token-list#22]
When you try loading https://cdn.jsdelivr.net/gh/solflare-wallet/token-list@latest/solana-tokenlist.json it crashes with an error `File size exceeded the configured limit of 20 MB.`. Apparently, the token list got too big to be handled by jsdelivr (default) limits.

**Comment by skakac** [comment]
Yes this is true, our use-case changed and we added more token providers.

Token list is still available via public / community API endpoints.

If anyone needs to use CDN, try using some other public CDN that will support the size of new token list.
